include/seqan3/contrib/charconv/charconv_pre.hpp

```diff
@@ -0,0 +1,57 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#pragma once
+
+#include <cassert>
+#include <system_error>
+#include <limits>
+
+#include <seqan3/core/platform.hpp>
+
+#define SEQAN3_CONTRIB_CHARCONV_LIBCPP_BEGIN_NAMESPACE_STD namespace seqan3::contrib::charconv {
```

Comment:
It's a little confusing that the macro name states "begin std namespace" but you are opening `seqan3::contrib::charconv`?

Reply:
true, but I want to keep the files as close to their original as possible.

Reply:
I removed them, because I don't need to define them.